test/API/endpoints/index.spec.js

```diff
@@ -0,0 +1,326 @@
+const assert = require('assert');
+
+// import classes to test
+const API = require('../../../API');
+const genStubAPIRequest = require('../helpers/stubAPIRequest');
+
+// import helper classes
+const utils = require('../../../EndpointCategory/helpers/utils');
+
+// Create API instance
+const testAPI = new API({
+  key: 'fakeKey',
+  secret: 'fakeSecret',
+  sendZoomRequest: genStubAPIRequest(),
+});
+
+// Set up the default date value to use for comparision later
+let defaultDate = new Date();
+defaultDate.setMonth(defaultDate.getMonth() - 6);
+defaultDate = utils.formatDate(defaultDate);
```

Comment:
I recommend setting this up in the test because other tests (not your own) might take lots of time so there might be a large delay between this code running and the test code running.